test/docfx.E2E.Tests/DocfxSeedSiteTest.cs

```diff
@@ -281,8 +281,7 @@ public void TestXRefMap()
         private void TestPageCommon()
         {
             // check logo
-            IWebElement element = _driver.FindElement(By.Id("logo"));
-            Assert.Equal("svg", element.TagName);
+            Assert.NotEmpty(_driver.FindElements(By.Id("logo")));
```

Comment:
>FindElements [](start = 36, length = 12)

question: the possible failure is caused by findElement or tag name? If not empty, will it fail by tage name != 'svg'?